_drafts/wework-com-is-going-static.markdown

```diff
@@ -0,0 +1,113 @@
+
+```coffeescript
+extensions: [
+  records(
+    marketsByCountry: {
+      url:
+        path: "#{process.env.DUBS_API}/api/v1/locations/markets_by_country"
+        headers: { Authorization: "Token token=#{process.env.DUBS_API_TOKEN}" }
+      template: "views/_location.jade"
+      out: (location) -> "/locations/#{location.market.slug}/#{location.slug}"
+    }
+  )
+]
+```
+
+
+### The static host with the most
+
+One of the biggest challenges of this whole process has been to figure out a way to slowly migrate our site over from a dynamic app, over to a static one. One way to do this would be to stop all development for a few months and rebuild the entire site using this new infrastructure. But as the saying goes, "ain't nobody got time for that."
+
+We needed a way to move things over piecemeal. One page at a time. This meant that we couldn't move wework.com to a new static host all at once, but at the same time, we needed some URL's to serve up the old content, and some URL's to serve up the new static pages. One way to do this is via a [reverse proxy](https://en.wikipedia.org/wiki/Reverse_proxy).
+
+There are many different ways to skin this cat. You can install and configure your own Apache or Ngnix server. You can use something like [`rack-reverse-proxy`](https://github.com/jaswope/rack-reverse-proxy) or a slew of other similar solutions. There are pros and cons to each approach. For us, being a team of web developers, we wanted to spend our time focusing on our KPI's and optimizing our pages for performance, and not on setting up and managing servers. We had a few requirements. Specifically, we were looking for a host that could:
+
+- House our new static site and scale with us
+- Have an API and webhooks so we can trigger builds when content changes
+- Ability to proxy requests to other URL's (internal and external)
+- Global CDN to improve our international traffic and SEO
+- Easy to use and configure
+- Great and responsive customer support
+
+We found one host that met all of those requirements and more, and they are called [Netlify](https://www.netlify.com/). Never heard of them? Neither had we, but as one engineer I recently spoke with who was familiar with their services put it, "Netlify is like the developer whisperer". This team has put together an amazing service that handles SO much for you from a dev-ops persective of hosting static sites. And if there is a feature that is missing, they will bend over backwards to either implement it for you, or help you figure out a solution. I can sit here and sing their praises all day long, but its probably best if I explained a bit about how they were able to help us with our migration to a static site and make the static verions of our site *2-3x faster*.
```

Comment:
**as one engineer I recently spoke with who was familiar with their services put it**, kind of wonky grammar 

maybe say a friend of mine? a colleague?